libs/utils.js

```diff
@@ -135,3 +137,17 @@ module.exports.handleBadToken = promise => {
     }
   })
 }
+
+module.exports.migrationDiff = (current, updated) => {
+  return jdiff(current, updated)
+    .replace('Received', 'Updated')
+    .replace('Expected', 'Current')
+}
```

Comment:
FYI the lib/mutations.js when displaying mutations accept `original` in its options and automatically shows the documents diffs. It is very recent ;)

Reply:
ok, good to know!
I guess I don't have to change this for this particular migration since we need to run it asap